lib/rubocop/cop/netlify/invalid_model_assignment.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Netlify
+      # This cop checks attribute assignment of Mongoid models
+      #
+      # @example
+      #   # bad
+      #   form.attributes[:email] = "bettse@netlify.com"
```

Comment:
just so that I understand; reading is okay (e.g. `form.attributes[:email]` -- well I guess still not preferred, but it still works), but writing is bad?

Reply:
Exactly.  Worth noting that the Mongoid attributes is documented as readonly (https://www.rubydoc.info/github/mongoid/mongoid/Mongoid/Attributes#attributes-instance_method)